brutils/boleto.py

```diff
@@ -0,0 +1,68 @@
+from brutils.types.boleto import Boleto
+
+
+def format_boleto(boleto: Boleto) -> str:
+    """
+     This function takes information from a boleto
+     and turns it into a string.
+
+     **Args:**
+     boleto (Boleto): A dictionary with boleto information
```

Comment:
```suggestion
     bank_slip: A string with the bank slip information.
```

Como uma boa prática de documentação, utilize pontos no final de cada linha.